test/integration/cli/cli-test.js

```diff
@@ -163,7 +163,7 @@ describe('CLI', () => {
 
         after(done => killAll('test/fixtures/scripts/', done));
 
-        it('should return with status 1', () => assert.equal(runtimeInfo.dredd.exitStatus, 1));
+        xit('should return with status 1', () => assert.equal(runtimeInfo.dredd.exitStatus, 1));
```

Comment:
Wow, didn't know about this one. I thought it's `it.skip(`

Reply:
Probably older API, but still works 🙂